src/us/kbase/workspace/test/database/provenance/RelationshipTypeTest.java

```diff
@@ -0,0 +1,71 @@
+package us.kbase.workspace.test.database.provenance;
+
+import org.junit.Test;
+
+import us.kbase.workspace.database.provenance.RelationshipType;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import us.kbase.common.test.TestCommon;
+
+import static us.kbase.workspace.test.database.provenance.ProvenanceTestCommon.WHITESPACE_STRINGS_WITH_NULL;
+
+public class RelationshipTypeTest {
+
+	@Test
+	public void testGetPid() throws Exception {
+		final RelationshipType rt = RelationshipType.OBSOLETES;
+		assertThat("incorrect PID",
+				rt.getPid(),
+				is("DataCite:Obsoletes"));
+	}
```

Comment:
No test / assertion for `getSource()`?

Reply:
`getSource` is private -- `getPid` should be used to retrieve output values from an instance.

Reply:
so it is - is `getSource` necessary in that case? Looks like it's only called in the map setup and seems to me like the instance variable could just be accessed directly